kit/file_watcher.go

```diff
@@ -156,9 +156,12 @@ func convertFsEvents(events chan fsnotify.Event, filter EventFilter) chan AssetE
 		recordedEvents := map[string]fsnotify.Event{}
 		for {
 			currentEvent = <-events
+			if currentEvent.Op&fsnotify.Chmod != fsnotify.Chmod {
+				recordedEvents[currentEvent.Name] = currentEvent
+			}
 			select {
 			case currentEvent = <-events:
-				if currentEvent.Op == fsnotify.Chmod {
+				if currentEvent.Op&fsnotify.Chmod == fsnotify.Chmod {
```

Comment:
Yes it is a bitwise AND, because the OP can be several different ops. You need to check it this way to be acurate. Check this out: https://github.com/fsnotify/fsnotify/blob/master/fsnotify.go#L37